libs/core-ui/src/lib/util/generateDefaultChartAxes.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  ChartTypes,
+  IGenericChartProps,
+  ISelectorConfig,
+  OtherChartTypes
+} from "./IGenericChartProps";
+import { JointDataset } from "./JointDataset";
+
+export function generateDefaultChartAxes(
+  jointDataset: JointDataset,
+  chartType: ChartTypes | OtherChartTypes
+): IGenericChartProps | undefined {
+  if (!jointDataset.hasDataset) {
+    return;
+  }
+  const chartProps: IGenericChartProps = {
+    chartType: chartType ? chartType : ChartTypes.Scatter,
+    colorAxis: {
+      options: {},
```

Comment:
Is `options` needed here?

Reply:
Yes, in ISelectorConfig interface option is an required param so it is defined as empty object.